themes/digital.gov/src/scss/new/styles.scss

```diff
@@ -128,6 +128,7 @@
 @forward "guides/guide-menu-bar";
 @forward "guides/guide-nav";
 @forward "guides/guide-prev-next";
+@forward "guides/guides";
```

Comment:
i wonder if this should be guides_landing to be more descriptive

Reply:
Agreed, perhaps `guides/guide-landing`.